### ISSUE TITLE
Module 3 Variant Validator LOVD 

### ISSUE THREAD

**Issue by SonjaR-UniMan** [gl186/SprintFinish#5]
Use HGVS genomic variant description as input and get MANE and MANE+Clinical variant description as output from VariantValidator using the requests module.

**Comment by SonjaR-UniMan** [comment]
First step: simple python script fulfilling in- and output criteria. 
Second step: create an endpoint to display in our own API to use this module on its own. 
Input: HGVS genomic variant description, output a json array containing all the MANE and MANE Select Plus data for the variant.

**Comment by SonjaR-UniMan** [comment]
json output translated to Python dict  to better rework, sort, filter data according to needs.

**Comment by ayurahman** [comment]
> Fourth step: Module automated testing added to tests folder. 

looking great

**Comment by ayurahman** [comment]
> First step: simple python script fulfilling in- and output criteria. 
> Second step: create an endpoint to display in our own API to use this module on its own. 
> Input: HGVS genomic variant description, output a json array containing all the MANE and MANE Select Plus data for the variant.

yup a working API without a code that pull the correct data will cause problem for the user
